eu_dgc_v1_schema.json

```diff
@@ -40,7 +61,7 @@
                         "properties": {
                             "t": {
                                 "title": "Identifier type",
-                                "description": "The type of identifier (viz. VS-2021-04-08) PPN = Passport NNxxx = national Person Identifier (ISO 3166-1 alpha-3 country code) CZ = Citizenship card HC = Health Card number etc.",
+                                "description": "The type of identifier (viz. VS-2021-04-14) PPN = Passport NNxxx = national Person Identifier (ISO 3166-1 alpha-3 country code) CZ = Citizenship card HC = Health Card number etc.",
```

Comment:
Didn't we want to remove the country code from the national person identifier? @jschlyter

Reply:
Country code in national identifier based on "Value Sets for the digital green certificate as stated in the Annex ...", abbr. "VS-2021-04-14" - https://webgate.ec.europa.eu/fpfis/wikis/x/05PuKg

Reply:
Keeping the ID type and country together has two major disadvantages:
1. Verifiers must parse them before handling NNSWE and NNFIN
2. We will not yet an issuing country on _other_ forms of id, e.g. passports (which is important)